dockerfiles/jupyter/README.ipynb

```diff
@@ -8,29 +8,71 @@
+    "    * Projects Documentation - [link](https://docs.mlrun.org/en/stable/projects/project.html)\n",
+    "    * Functions Documentation - [link](https://docs.mlrun.org/en/stable/runtimes/functions.html)\n",
+    "    * Artifacts Documentation - [link](https://docs.mlrun.org/en/stable/store/artifacts.html)\n",
+    " * Quick start tutorial for machine learning tutorial - [link](https://docs.mlrun.org/en/stable/tutorials/01-mlrun-basics.html)\n",
+    " * MLRun cheat sheet - [link](https://docs.mlrun.org/en/stable/cheat-sheet.html#mlrun-setup)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ddeb6a5c",
+   "metadata": {},
+   "source": [
+    "### MLRun CE default components\n",
     "\n",
-    "This Jupyter Notebook server is designed to work with the different options, it works out of the box with the **local** deployment mode.<br> \n",
-    "In order to work with remote MLRun service (Kubernetes or managed) you need to edit and save the [**mlrun.env**](./mlrun.env) file in the following way:\n",
+    "The MLRun CE (Community Edition) chart includes the following components below by default:\n",
+    "* MLRun - https://github.com/mlrun/mlrun\n",
+    "  - MLRun API\n",
+    "  - MLRun UI\n",
+    "  - MLRun DB (MySQL)\n",
+    "* Nuclio - https://github.com/nuclio/nuclio\n",
+    "* Jupyter - https://github.com/jupyter/notebook (+MLRun integrated)\n",
+    "* MPI Operator - https://github.com/kubeflow/mpi-operator\n",
+    "* MinIO - https://github.com/minio/minio/tree/master/helm/minio\n",
+    "* Spark Operator - https://github.com/GoogleCloudPlatform/spark-on-k8s-operator\n",
+    "* Pipelines - https://github.com/kubeflow/pipelines\n",
+    "* Prometheus stack - https://github.com/prometheus-community/helm-charts\n",
+    "  - Prometheus\n",
+    "  - Grafana"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7aeb1ea8",
+   "metadata": {},
+   "source": [
+    "### MLRun CE Addional components \n",
     "\n",
-    "```sh\n",
-    "# set remote MLRun service address, username and access-key\n",
-    "MLRUN_DBPATH=https://<service-address>\n",
-    "V3IO_USERNAME=<user>\n",
-    "V3IO_ACCESS_KEY=<access-key>\n",
-    "```\n",
+    "There are a couple of default components enabled by default within the MLRun CE installation (see the list above). \n",
+    "The list below describes additional components that you can enable if needed - \n",
+    "\n",
+    "* MLRun Model monitoring requires the installation of the two applications below, [see also](https://docs.mlrun.org/en/stable/model-monitoring/index.html#selecting-the-streaming-and-tsdb-platforms).\n",
+    "    * [TDEngine](https://taosdata.github.io/TDengine-Operator/en/2.2-tdengine-with-helm.html) (3.3.2.0) \n",
+    "    * [Kafka](https://github.com/bitnami/charts/tree/main/bitnami/kafka) (3.2.3)\n",
+    "* Real Time feature store, requires [Reddis](https://redis.io/docs/latest/operate/oss_and_stack/install/) installation as the feature store real time db (KV).\n",
```

Comment:
"* Real Time feature store: requires [Reddis](https://redis.io/docs/latest/operate/oss_and_stack/install/) installation as the feature store real time db (KV).\n",
     "    See also [Configuring the online feature store](https://docs.mlrun.org/en/stable/install/kubernetes.html#configuring-the-online-feature-store)\n",